docker-compose.yml

```diff
@@ -0,0 +1,6 @@
+xbundle:
+  build: .
+  volumes:
+    - .:/src
+    - /tmp
```

Comment:
I probably put it there because I was using it for easy access to logs. It _could_ be useful for examining the data dumped to temp dirs during tests, but at this point we're not troubleshooting tests.

It's fine to stay or go; it doesn't really matter. Just curious.